fundamentals/problem-set-fundamentals-vocabulary.md

```diff
@@ -617,6 +639,13 @@ Which of the following are literals?
 * `[]`
 
 ##### !end-answer
+##### !explanation
```

Comment:
I didn't add a hint here because this question is the same as the question above but with different options. I can add the same hint as what is shown above on line 599 if it feels like this question should also have a hint.

Reply:
I think the hint from above is enough to suffice!